manifests/init.pp

```diff
@@ -28,6 +30,8 @@
 class kibana (
   Variant[Enum['present', 'absent', 'latest'], Pattern[/^\d([.]\d+)*(-[\d\w]+)?$/]] $ensure,
   Hash[String[1], Variant[String, Integer, Boolean, Array, Hash]] $config,
+  String $kibana_user,
+  String $kibana_group,
```

Comment:
An empty string is not valid so let's put a minimum size. Also set the default here instead of in Hiera, that way the reference file can tell us what is the default value.

```suggestion
  String[1] $kibana_user  = 'kibana',
  String[1] $kibana_group = 'kibana',
```

Reply:
ok